app/views/ems_block_storage/show_list.html.haml

```diff
@@ -1 +1,7 @@
-#main_div= render :partial => "layouts/gtl"
+#main_div
+  - if ManageIQ::Providers::StorageManager.any?
+    = render :partial => 'layouts/gtl'
+  - else
+    = render :partial => 'layouts/empty',
+             :locals => {:add_message => _("Add a new Block Storage Manager"),
+                         :documentation => ::Settings.docs.product_documentation_website} # [todo] we need to add storage_provider docs and set the URL for it in manageiq/config/settings.yml
```

Comment:
You should be able to use `::Settings.docs.storage_manager` after this is merged https://github.com/ManageIQ/manageiq/pull/20638

Reply:
done